ccd-definition/ComplexTypes/CareSupervision/GeneratedOrder.json

```diff
@@ -4,7 +4,7 @@
     "ID": "GeneratedOrder",
     "ListElementCode": "type",
     "FieldType": "Text",
-    "ElementLabel": "Type of order",
+    "ElementLabel": "Order type",
```

Comment:
is an old or new field?

Reply:
It's an old field, I can see it was added back in 2019. This probably isn't a necessary change so I will revert this